src/Sylius/Bundle/PromotionBundle/Resources/config/services/command.xml

```diff
@@ -0,0 +1,11 @@
+<container xmlns="http://symfony.com/schema/dic/services">
+    <services>
+        <defaults public="true" />
+
+        <service class="Sylius\Bundle\PromotionBundle\Command\GenerateCouponsCommand"
```

Comment:
Id `Sylius\Bundle\PromotionBundle\Command\GenerateCouponsCommand` would be enough 👌 And don't we need a `console.command` tag?